README.md

```diff
@@ -505,22 +505,22 @@ openshift.withCluster( 'mycluster' ) {
 ### Who are you, really?
 Getting advanced? You might need more than just default credentials associated
 with your cluster. You can leverage any OpenShift Token credential type in the Jenkins
-credential store by passing doAs the credential's identifier. If you think
-security is a luxury you can live without (it's not), you can also pass doAs
+credential store by passing withCredentials the credential's identifier. If you think
+security is a luxury you can live without (it's not), you can also pass withCredentials
```

Comment:
we should have a quick blurb that explains doAs was deprecated (but still technically supported)

Reply:
updated in next push